### DISCUSSION TITLE
IMSI catcher

### DISCUSSION THREAD

**Discussion by Jakesdp1982** [Oros42/IMSI-catcher#135] [Q&A]
Hi Oros42, thank you very much for the this code and project, I find it amazing and scary at the same time.
I have used your guide and successfully built an IMSI Catcher, I just have a question about the device, will it get any IMSI numbers if there is no cell tower in range? So what I am asking, is if I am in the middle of the desert with no cell reception, but right next to the IMSI catcher, will it catch my cell phone's IMSI number?

Thank you very much in advance

Jaco

**Comment by Oros42** [comment]
Hello
No, because my program use gr-gsm who listen cell towers and not cell phone.

**Reply by Jakesdp1982** [reply]
Thank you for the reply.
So if I am correct, this IMSI Cather tells you if a phone is in the range of a cell tower?

Would it be possible for this Simple IMSI Catcher to be modified to talk to the cell phone instead? 
Could I mail or contact you privately? I would like to explain why I am enquiring about this. I wont take a lot of your time.

I would really appreciate it.

**Comment by Oros42** [comment]
It's not possible because I use gr-gsm.

**Reply by Jakesdp1982** [reply]
ok thank you.
Sorry for the 20 questions, but do you know of a way to be able to locate cell phones in an area that are not supposed to be there for example?
You seem to have a lot of knowledge on the subject, that is why I am asking you.
Thanks again

**Comment by iwantyou123** [comment]
First of all, thank you, you have a question:
i check my imsi, switch mobile to gsm mode, check cellid, then select grgsm_livemon correct frequency.
  But with the command "sudo python3 simple_IMSI-catcher.py -s" it can't find my IMSI, even though it shows a lot of IMSIs in the list.
Then, use it again with the command "sudo python3 simple_IMSI-catcher.py -s -m xxx", where xxx is the IMSI just found with the -s option, but this command also has no results.
You ask yourself why the search command by IMSI does not have such a response.
Thank you again and look forward to your feedback.
Thank you.
